vault-plugins/flant_iam_auth/io/kafka_handlers/root/object_handler_test.go

```diff
@@ -393,8 +396,10 @@ func TestRootMessageDispatcherCreate(t *testing.T) {
 			require.Equal(t, e.UserId, uuid)
 			require.Equal(t, e.Name, c.fullId(user), "must name same af full_id")
 
-			aliases, _ := getAllAliases(t, tx, uuid)
-			require.Len(t, aliases, 1, "should does create one aliase for internal multipass source")
+			aliases, e2 := getAllAliases(t, tx, uuid)
+			fmt.Printf("%#v", e2)
```

Comment:
Do we need print it here? Or, was it added for debug?

Reply:
fix